wicker/core/storage.py

```diff
@@ -52,36 +75,41 @@ def __init__(self) -> None:
         """Constructor for a file system storage instance."""
         pass
 
```

Comment:
Below is the main functionality change to the `FileSystemDataStorage` class for local file systems. We don't want to add any automatic retry / backoff / delay stuff for local filesystems since for GCP this is directly built into GCSFuse itself.